pkg/web/src/app/Providers/components/ProvidersTable/OpenShift/OpenShiftStorageClassList.tsx

```diff
@@ -0,0 +1,34 @@
+import { hasCondition } from '@app/common/helpers';
```

Comment:
This file is still being considered a delete/rewrite, can we make it a `git mv`?

Reply:
Initially it happened because a rebase. I thought I fixed it, weird. I'll check it in again.

Reply:
The file came out with lint issues. So it had to be fixed. The history is preserved